Tinylog/View Controllers/Lists/AddListViewController.swift

```diff
@@ -40,40 +41,52 @@
         super.viewDidLoad()
 
         setupNavigationBarProperties()
-
+        setupUITableView()
+        setupNavigationItem()
+        setupMenuColorsView()
+        setupViewMode()
+    }
+    
+    private func setupUITableView() {
+        tableView.isScrollEnabled = false
         tableView.backgroundColor = UIColor(named: "mainColor")
         tableView.separatorColor = UIColor(named: "tableViewSeparator")
-
+        tableView.register(TextFieldCell.self,
+                           forCellReuseIdentifier: TextFieldCell.cellIdentifier)
+    }
+    
+    private func setupNavigationItem() {
         navigationItem.leftBarButtonItem = UIBarButtonItem(title: "Cancel",
-                                                           style: UIBarButtonItem.Style.plain,
+                                                           style: .plain,
                                                            target: self,
                                                            action: #selector(AddListViewController.cancel(_:)))
 
         navigationItem.rightBarButtonItem = UIBarButtonItem(title: "Save",
-                                                            style: UIBarButtonItem.Style.plain,
+                                                            style: .plain,
                                                             target: self,
                                                             action: #selector(AddListViewController.save(_:)))
-
-        menuColorsView = MenuColorsView(frame: CGRect(x: 12.0, y: 200.0, width: view.frame.width, height: 51.0))
+    }
+    
+    private func setupMenuColorsView() {
+        menuColorsView = MenuColorsView(frame: CGRect(x: 12.0, y: 0.0, width: view.frame.width, height: 51.0))
+        menuColorsView?.delegate = self
         tableView.tableFooterView = menuColorsView
-
-        if mode == .create {
+    }
+    
```

Comment:
<table data-meta="generated_by_danger">
  <tbody>
    <tr>
      <td>:warning:</td>
      <td width="100%" data-sticky="false"><span data-href="https://github.com/sger/Tinylog/blob/03eace86b303bd47c13e99b11bd2b1214da5c26b/Tinylog/View Controllers/Lists/AddListViewController.swift#L75"></span>Lines should not have trailing whitespace.<br />
<code>trailing_whitespace</code> <code>AddListViewController.swift:75</code></td>
    </tr>
  </tbody>
</table>